apps/docs/content/guides/auth/auth-anonymous.mdx

```diff
@@ -56,6 +56,10 @@ For self-hosting, you can update your project configuration using the files and
 Call the [`signInAnonymously()`](/docs/reference/javascript/auth-signinanonymously) method:
 
 ```js
+import { createClient } from '@supabase/supabase-js'
+const supabase = createClient('https://your-project.supabase.co', 'your-anon-key')
+
+// ---cut---
```

Comment:
What does this mean?

Reply:
It hides everything above the cut (we need these statements in order to get the type hints, but we don't want to actually show them in the UI cause they're just clutter).

From the contributing guide:

<img width="877" alt="image" src="https://github.com/user-attachments/assets/dd84a786-2029-4f16-a813-ddf41b799ec2" />